manifests/base/commit-latest.env

```diff
@@ -1,20 +1,20 @@
-odh-pipeline-runtime-datascience-cpu-py312-ubi9-commit-n=ec4a477
-odh-pipeline-runtime-minimal-cpu-py312-ubi9-commit-n=ed2d978
-odh-pipeline-runtime-pytorch-cuda-py312-ubi9-commit-n=ec4a477
-odh-pipeline-runtime-pytorch-llmcompressor-cuda-py312-ubi9-commit-n=646aeb5
-odh-pipeline-runtime-pytorch-rocm-py312-ubi9-commit-n=ec4a477
-odh-pipeline-runtime-tensorflow-cuda-py312-ubi9-commit-n=ec4a477
-odh-pipeline-runtime-tensorflow-rocm-py312-ubi9-commit-n=93c0810
-odh-workbench-codeserver-datascience-cpu-py312-ubi9-commit-n=ec4a477
-odh-workbench-jupyter-datascience-cpu-py312-ubi9-commit-n=ec4a477
-odh-workbench-jupyter-minimal-cpu-py312-ubi9-commit-n=ec4a477
-odh-workbench-jupyter-minimal-cuda-py312-ubi9-commit-n=ec4a477
-odh-workbench-jupyter-minimal-rocm-py312-ubi9-commit-n=ec4a477
-odh-workbench-jupyter-pytorch-cuda-py312-ubi9-commit-n=ec4a477
-odh-workbench-jupyter-pytorch-llmcompressor-cuda-py312-ubi9-commit-n=ec4a477
-odh-workbench-jupyter-pytorch-rocm-py312-ubi9-commit-n=ec4a477
-odh-workbench-jupyter-tensorflow-cuda-py312-ubi9-commit-n=ec4a477
-odh-workbench-jupyter-tensorflow-rocm-py312-ubi9-commit-n=ec4a477
-odh-workbench-jupyter-trustyai-cpu-py312-ubi9-commit-n=ec4a477
-odh-workbench-rstudio-minimal-cpu-py311-c9s-commit-n=0d8752d
-odh-workbench-rstudio-minimal-cuda-py311-c9s-commit-n=0d8752d
+odh-pipeline-runtime-datascience-cpu-py312-ubi9-commit-n=564f037
+odh-pipeline-runtime-minimal-cpu-py312-ubi9-commit-n=dummy
+odh-pipeline-runtime-pytorch-cuda-py312-ubi9-commit-n=564f037
+odh-pipeline-runtime-pytorch-llmcompressor-cuda-py312-ubi9-commit-n=dummy
+odh-pipeline-runtime-pytorch-rocm-py312-ubi9-commit-n=dummy
+odh-pipeline-runtime-tensorflow-cuda-py312-ubi9-commit-n=dummy
+odh-pipeline-runtime-tensorflow-rocm-py312-ubi9-commit-n=564f037
+odh-workbench-codeserver-datascience-cpu-py312-ubi9-commit-n=dummy
+odh-workbench-jupyter-datascience-cpu-py312-ubi9-commit-n=564f037
+odh-workbench-jupyter-minimal-cpu-py312-ubi9-commit-n=564f037
```

Comment:
Why some entries are exceptions from the dummy group? 🤣

Reply:
it's better now, much fewer dummy entries